README.md

```diff
@@ -13,7 +13,7 @@
 
 A collection of simplified and community-driven man pages.
 
-Install it with `npm install -g tldr` or [try the web client](http://tldr.ostera.io).
+Install it with `npm install -g tldr`, `brew install tldr` (on OSX, using [Homebrew](http://brew.sh)), or [try the web client](http://tldr.ostera.io).
```

Comment:
I would refrain from mentioning this here, to keep the opening lines short and snappy. Mentioning it for the individual clients is fine.